guby_initialize.sh

```diff
@@ -22,16 +22,16 @@ echo "###########----##########";
 if [ "$INITIALIZE_DB" == true ]
 then
     echo "Initializing database for guby"
-    python manage.py makemigrations 
-    python manage.py migrate
+    yes | python manage.py makemigrations 
+    yes yes | python manage.py migrate
```

Comment:
As discussed, let's run a quick check with the `--no-input` flag passed before we use `yes`

Reply:
I cannot check this now, can you?